platforms/intellij/skate/build.gradle.kts

```diff
@@ -118,39 +117,20 @@ configurations
   .configureEach { attributes { attribute(KotlinPlatformType.attribute, KotlinPlatformType.jvm) } }
 
 dependencies {
-  lintChecks(libs.composeLints)
-
-  compileOnly(libs.coroutines.core.ij)
+  implementation(projects.platforms.intellij.compose, exclusions)
+  implementation(projects.tools.tracing, exclusions)
 
   implementation(libs.bugsnag)
-  implementation(libs.okio)
   implementation(libs.kaml)
   implementation(libs.kotlinx.serialization.core)
   implementation(libs.okhttp)
   implementation(libs.okhttp.loggingInterceptor)
-  implementation(projects.platforms.intellij.compose, exclusions)
-  implementation(projects.tools.tracing, exclusions)
+  implementation(libs.okio)
 
-  intellijPlatform {
```

Comment:
why is this block removed?

Reply:
see below :). Looks like it's a bug

Reply:
[fbc1758](https://github.com/slackhq/foundry/pull/1027/commits/fbc1758069043f7571d9744c33b31510036cd1fb)